dciv_bot/module/misc.py

```diff
@@ -579,6 +580,85 @@ async def invite(self, ctx):
         invite = await ctx.channel.create_invite(max_age=0, unique=False)
         await ctx.send(invite.url)
 
+    @commands.command(name='roll')
+    @commands.cooldown(1, config.BOT_COMMAND_COOLDOWN, commands.BucketType.user)
+    async def roll(self, ctx, *args):
```

Comment:
```suggestion
    async def roll(self, ctx, *, dices):
```

Putting all the arguments in one string, aka "consume rest behaviour", is done like this:

`async def roll(self, ctx, *, args):`

Reply:
Maybe make the argument name more human sounding? Like `dices`, `dice` or something along the lines of that.